templates/taxi/car_confirm_delete.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+
+{% block content %}
+    <h1>Delete Car</h1>
+    <p>Are you sure want to delete the car?</p>
+
+    <form action="" method="post">
+        {% csrf_token %}
+        <input type="submit" value="Yes, delete" class="btn btn-danger" />
```

Comment:
it will be a good option for the user to cancel deletion , that’s why we have an extra step before deleting something